modules/combined/test/tests/restart-transient-from-ss-with-stateful/master_tr.i

```diff
@@ -0,0 +1,119 @@
+[Problem]
+  restart_file_base = master_ss_checkpoint_cp/LATEST
+  force_restart = true
+  skip_additional_restart_data = true
+[]
+
+[Mesh]
+  file = master_ss_checkpoint_cp/LATEST
```

Comment:
Not related to this PR. It would be better if we need to set the file name in one place (not two).

`Problem` might take the file name in `Mesh`